django/db/models/functions/math.py

```diff
@@ -141,6 +207,17 @@ def as_oracle(self, compiler, connection, **extra_context):
         )
 
 
+class Random(NumericOutputFieldMixin, Func):
```

Comment:
This already exists in `db/models/expressions.py`, for `order_by('?')`, but is undocumented. Perhaps it should be moved. The backends already implement `random_function_sql` to support this.

Reply:
So we can simply ditch `DatabaseOperations.random_function_sql()` which is a relic of the past from before expressions and functions were supported.

The only difference is making the following changes to make the function `Random` consistent:
- `DBMS_RANDOM.VALUE` -- values in  `[0, 1)` -- over `DBMS_RANDOM.RANDOM` -- values in `[-2^31, 2^31)` -- for Oracle
- Adding custom function using [`random.random()`](https://docs.python.org/3/library/random.html#random.random) instead of built-in `RANDOM` -- value in `[-2^63, 2^63)` -- for SQLite.

These changes should have no negative impact on the use of `'?'` for ordering.

Reply:
👍 

This should have a release note for third party DB backends to pick up on.

Reply:
[:heavy_check_mark:  Done](https://github.com/django/django/pull/13252/commits/c8913d41e74859747893dc21356a1fb6906c56e9#diff-11891657485a3374485fd93545354ec6R380-R381)